src/Domain/Structure/Neighbors.hpp

```diff
@@ -31,8 +31,22 @@ class Neighbors {
                 std::is_same_v<IdType, ElementId<VolumeDim>>);
 
  public:
+  /// Construct with the ids, the orientation of the neighbors relative to the
+  /// host, and whether the neighbors are conforming.
+  ///
+  /// \param ids the ids of the neighbors.
+  /// \param orientations An OrientationMap (which takes objects in the logical
+  /// coordinate frame of the host and maps them to the logical coordinate frame
+  /// of the neighbor) for each neighboring Block (Elements within a Block share
+  /// the same orientation).
+  /// \param are_conforming whether or not the block logical coordinates of the
+  /// neighbors conform to those of the host
```

Comment:
Could you add examples of conforming and not conforming? This way it's abundantly clear

Reply:
can I point to the documentation of neighbor_is_conforming or do you want something else?

Reply:
That's fine :) Just so that future-me can avoid fuming about you ;)